nativescript-angular/directives/dialogs.ts

```diff
@@ -159,7 +177,13 @@ export class ModalDialogService {
 
             // TODO: remove <any> cast after https://github.com/NativeScript/NativeScript/pull/5734
             // is in a published version of tns-core-modules.
-            (<any>parentView).showModal(componentView, context, closeCallback, fullscreen, animated, stretched);
+            if (ios) {
```

Comment:
As already mentioned, the code in the else is being deprecated and is not longer needed. You could simply do:

 ```
           let modalOptions: ShowModalOptions = {
                closeCallback: closeCallback,
                context: context,
                fullscreen: fullscreen,
                animated: animated,
                stretched: stretched,
                ios: ios
            };
             // TODO: remove <any> cast after https://github.com/NativeScript/NativeScript/pull/5734
             // is in a published version of tns-core-modules.
            (<any>parentView).showModal(componentView, modalOptions);
```

Reply:
I've made the changes but based it on this PR https://github.com/NativeScript/nativescript-angular/pull/1769 as I think that's a better way to do it and will mean we don't have to keep adding options to match the core.